### PR TITLE
Feature issue 4981

### PR DESCRIPTION
**Pull request by ranj8work**
[vagrant][registry] Changes to fix the issue 4981 

## Summary
The changes are made in orc8r/tools/ansible/roles/pkgrepo/tasks/debian.yml and redhat.yml files to fix the Issue 4981 (https://github.com/magma/magma/issues/4981), to download the gpg keys from JFROG repo.

## Test Plan
Mocked the circleci config file to force build and test using lte-integ-test job under the agw workflow.

## Additional Information

- [ NO] This change is backwards-breaking

### CODE REVIEW

File: orc8r/tools/ansible/roles/pkgrepo/tasks/debian.yml [hunk 1]
```diff
@@ -23,9 +23,15 @@
   #  when: preburn
 
 - name: Add JFrog key
-  apt_key:
-    data: "{{ lookup('file', 'jfrog.pub') }}"
-    state: present
+  block:
```
Comment (tmdzk) [inline orc8r/tools/ansible/roles/pkgrepo/tasks/debian.yml]: As discuss, I think we can do it one shot with apt_key and specifying the url. [ansible module for apt](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_key_module.html) 

```bash
- name: Add an Apt signing key, uses whichever key is at the URL
  ansible.builtin.apt_key:
    url: https://ftp-master.debian.org/keys/archive-key-6.0.asc
    state: present
```

File: orc8r/tools/ansible/roles/pkgrepo/tasks/debian.yml [hunk 2]
```diff
@@ -23,9 +23,15 @@
...
+    - get_url:
```
Comment (tmdzk) [inline orc8r/tools/ansible/roles/pkgrepo/tasks/debian.yml]: Pretty sure you didn't have to store it in a variable and you could have just used `file:` in the apt_key command

**Review by tmdzk — CHANGES_REQUESTED** [review]
Please fill the 
```
Summary
Test Plan
Additional Information
[ NO] This change is backwards-breaking
```

and give a more explicit names

**Review by tmdzk — APPROVED** [review]

**Review by hcgatewood — APPROVED** [review]
Don't have context for this workflow, deferring to Tim's stamp

**Pull request merged**